go.mod

```diff
@@ -87,7 +87,7 @@ require (
 	go.uber.org/zap v1.24.0
 	go.viam.com/api v0.1.322
 	go.viam.com/test v1.1.1-0.20220913152726-5da9916c08a2
-	go.viam.com/utils v0.1.87
+	go.viam.com/utils v0.1.88
```

Comment:
@dgottlieb note that [this update](https://github.com/viamrobotics/goutils/releases/tag/v0.1.88) will include our "swallowing" patch too.